docs/source/column.rst

```diff
@@ -10,6 +10,7 @@ Column
     :toctree: api/
 
     Column
+    CaseExpr
```

Comment:
Do we want to sort it alphabetically? Similar to the methods below.

Reply:
CaseExpr is a subclass of Column so I think placing it after is better.